static/src/javascripts/projects/common/views/commercial/creatives/fluid250.html

```diff
@@ -9,6 +9,7 @@
     ">
     <a href="{{clickMacro}}{{link}}" target="_blank">
         <div class="gs-container">
+            {{videoD}}
```

Comment:
Good spot, D was there when I was going to be using video in mobile version as well which we aren't doing any more.